### PR TITLE
Develop hero section for mobile version

### PR DESCRIPTION
**Pull request by zainsadaqat**
## In this pull request, I've done the following tasks:

I created Index.html and Style.css files and Images folder that contain all the social media icons in SVG formats and Background Image in PNG format, where Index.html has the following items:

> Header Section
* Logo
* Hamburger Icon

> Banner Section ( Hero Section )
* Heading
* Paragraph
* Caption
* Social Media Links

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Project invalidated 🚫 

Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

cat_massage

## Why does this happen ❓ ❓ ❓ 

You have submitted a project, that actually has a readme file template, but you don't have added info for your project there, I understand to follow the template is important, but you need to change the info inside according to your project. I share to you [this example](https://github.com/vichuge/react-calculator/blob/master/README.md) to understand better what you need to do.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zainsadaqat** [comment]
I'm really sorry for that, I completely forgot to update the README file.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        </p>
+        <h3 class="hero-caption">Let's Connect</h3>
+        <div class="hero-social-media">
+          <img src="/images/Twitter.svg" alt="Twitter" />
+          <img src="/images/LinkedIn-Icon.svg" alt="LinkedIn" />
+          <img src="/images/Medium.svg" alt="Medium" />
+          <img src="/images/Github.svg" alt="Github" />
+          <img src="/images/Victory.svg" alt="Victory" />
+        </div>
+      </section>
```
Comment (arslanbisharat) [inline index.html]: - [x] This section should enclosed in ul and li tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <img src="/images/logo.svg" alt="Logo" />
+      <img src="/images/menu-icon.svg" alt="Menu Icon" />
+    </header>
+    <main class="hero-section">
+      <section>
+        <h1 class="hero-heading">I'm Zain Glad to see you!</h1>
```
Comment (arslanbisharat) [inline index.html]: - [x] Subtitle line should use hx

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @zainsadaqat,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (zainsadaqat) [reply index.html]: I'm sorry, I didn't understand what hx is. Can you please explain?

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @zainsadaqat , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️

- [x] please wrap your social link icons with an `a` tag , the `img` tag can be inside of the `a` tag so that the links are clickable

### Info
`hx` means any of the `h` tags from 2-6  like `h2, h3 or h4`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔✔**
Hello @zainsadaqat 👋🏽
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding! 👏🏽👨🏽‍💻👏🏽

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zainsadaqat** [comment]
# Thanks, @emmanuellekamwa for approving this pull request and @vichuge @arslanbisharat @AshakaE for pointing out my mistakes to make me a good developer. Thanks to all

**Pull request merged**